xchange-stream-btcmarkets/src/main/java/info/bitrich/xchangestream/btcmarkets/service/BTCMarketsStreamingService.java

```diff
@@ -0,0 +1,70 @@
+package info.bitrich.xchangestream.btcmarkets.service;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import info.bitrich.xchangestream.btcmarkets.dto.BTCMarketsWebSocketSubscribeMessage;
+import info.bitrich.xchangestream.service.netty.JsonNettyStreamingService;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BTCMarketsStreamingService extends JsonNettyStreamingService {
```

Comment:
Does this class itself need to be `public`?

Reply:
I've moved things out of the packages and removed  `public`.